app/src/main/java/org/mozilla/fenix/components/metrics/MetricController.kt

```diff
@@ -213,16 +216,16 @@ internal class ReleaseMetricController(
         Component.SUPPORT_WEBEXTENSIONS to WebExtensionFacts.Items.WEB_EXTENSIONS_INITIALIZED -> {
             metadata?.get("installed")?.let { installedAddons ->
                 if (installedAddons is List<*>) {
-                    Addons.installedAddons.set(installedAddons.map { it.toString() })
-                    Addons.hasInstalledAddons.set(installedAddons.size > 0)
+                    settings.installedAddonsCount = installedAddons.size
+                    settings.installedAddonsList = installedAddons.joinToString(",")
                     Leanplum.setUserAttributes(mapOf("installed_addons" to installedAddons.size))
                 }
             }
 
             metadata?.get("enabled")?.let { enabledAddons ->
                 if (enabledAddons is List<*>) {
-                    Addons.enabledAddons.set(enabledAddons.map { it.toString() })
-                    Addons.hasEnabledAddons.set(enabledAddons.size > 0)
+                    settings.enabledAddonsCount = enabledAddons.size
+                    settings.enabledAddonsList = enabledAddons.joinToString()
```

Comment:
This change breaks some AMO stats as well as some Redash dashboards because the default value when joining items is `, ` (with a trailing space). The ETL stuff expects a comma-separated list without spaces (we don't trim items after `split`).

Reply:
Issue created: https://github.com/mozilla-mobile/fenix/issues/19289